### PR TITLE
chore: Replacing Juju bundles with Terraform modules in the CUPS how to

### PR DESCRIPTION
**Pull request by Gmerold**
# Description

Replaces the Juju bundles with Terraform in the CUPS tutorial

# Checklist:

- [ ] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that validate the behaviour of the software
- [ ] I validated that new and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have bumped the version of the library

### CODE REVIEW

File: docs/how-to/deploy_sdcore_cups.md [hunk 1]
```diff
@@ -8,70 +8,237 @@ This guide covers how to install a SD-Core 5G core network with Control Plane an
...
+  model            = "control-plane"
+  application_name = module.<CONTROL PLANE MODULE NAME>.amf_app_name
+  endpoint         = module.<CONTROL PLANE MODULE NAME>.fiveg_n2_endpoint
+}
+
+resource "juju_integration" "fiveg-n2" {
+  model = "control-plane"
+
+  application {
+    name     = <THE `fiveg-n2` REQUIRER APP>
```
Comment (gatici) [inline docs/how-to/deploy_sdcore_cups.md:109]: We do not mention about the deployment of fiveg-n2 requirer app (Gnbsim). It seemed to me that a piece is missing here it users only checks this document. We know that it's included in another how-to. What about giving a reference to  Gnbsim deployment how-to guide.

File: docs/how-to/deploy_sdcore_cups.md [hunk 2]
```diff
@@ -8,70 +8,237 @@ This guide covers how to install a SD-Core 5G core network with Control Plane an
...
+resource "juju_integration" "fiveg-n4" {
+  model = "user-plane"
+
+  application {
+    name     = module.upf.app_name
+    endpoint = module.upf.fiveg_n4_endpoint
+  }
+
+  application {
+    name     = <THE `fiveg_n4` REQUIRER APP>
```
Comment (gatici) [inline docs/how-to/deploy_sdcore_cups.md:193]: In general, fiveg_n4 requirer could be an external app. However we already deployed one in our sdcore-control plane which is `NMS`. Could we mention from our `NMS` as a sample requirer app which could be integrated as a preference.

File: docs/how-to/deploy_sdcore_cups.md (hunk 1 above)
Comment (Gmerold) [reply docs/how-to/deploy_sdcore_cups.md:109]: The `fiveg-n2` requirer is not necessarily gNB simulator. We won't be using it at all in the real deployments. I don't think that the how-to guide should focus on this aspect.

File: docs/how-to/deploy_sdcore_cups.md (hunk 2 above)
Comment (Gmerold) [reply docs/how-to/deploy_sdcore_cups.md:193]: Similarly as above, I don't think that the how-to guide is the right place for this kind of examples.

**Review by gatici — APPROVED** [review]

**Pull request merged**